hedera-node/test-clients/build.gradle.kts

```diff
@@ -46,6 +46,8 @@ sourceSets {
     // Needed because "resource" directory is misnamed. See
     // https://github.com/hashgraph/hedera-services/issues/3361
     main { resources { srcDir("src/main/resource") } }
+
+    create("rcdiff") { java.srcDir("src/rcdiff/java") }
```

Comment:
@jjohannes Can this be simplified at all or the hardcoded path specification removed?

Reply:
```suggestion
    create("rcdiff")
```

Yes. We do not need the `srcDir` part as this this is the default.

Reply:
Removed.